### PR TITLE
chore(guide): update index.md

### PR DESCRIPTION
**Pull request by OneRadRedDad**
# Pull Request

## Purpose

To update grammar and improve readability of index.md

## Requirements

- [X] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [X] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by TRaSH-** [comment]
Thnx, i will let our new own grammar control review this one.
not my place to argue about English grammar

File: docs/index.md [hunk 1]
```diff
@@ -16,34 +16,34 @@ Here you will find guides mainly for Sonarr/Radarr/Bazarr and everything related
...
-So I decided to fine tune it to try to get the releases & quality I preferred. On Discord, I also noticed that others run in to the same issue so I explained them how I managed to get it solved. Being that I had to explain it often, I decided to write it down and make it public so I could easily link it to someone when they needed it.
-After a while I also decided to create guides for frequently asked questions, and started to collect all kind of information gathered from various sources.
-I try to make my guides as easy as possible for everyone to understand, in my opinion pictures explain often more than just some text.
+When I started using Sonarr/Radarr, I noticed that it didn't grab the releases that I actually wanted (I'm a bit picky about which releases and what quality I prefer).
+I decided to fine tune my quality profile settings to try to get the releases & quality I preferred. On Discord, I noticed that others ran in to the same issue, so I explained how I managed to get it solved. I started having to explain it often, so I decided to write it down and make it public so I could easily link it to someone when they needed it.
```
Comment (zakkarry) [inline docs/index.md:20]:
```suggestion
I decided to fine-tune my quality profile settings to try and get the releases and qualities I preferred. While on Discord, I noticed that others ran into the same issue, so I explained how I managed to solve this. Once I started having to explain more often, I decided to write it down and make it public so I could easily link to it when needed.

File: docs/index.md [hunk 2]
```diff
@@ -16,34 +16,34 @@ Here you will find guides mainly for Sonarr/Radarr/Bazarr and everything related
...
+After a while I *also* decided to create guides for frequently asked questions and started to collect all kinds of information gathered from various sources.
+I try to make my guides as easy as possible for everyone to understand. In my opinion, pictures often explain more than just some text.
```
Comment (zakkarry) [inline docs/index.md:22]:
```suggestion
I try to make my guides as easy as possible to understand. In my opinion, pictures often explain more than just some text.

File: docs/index.md [hunk 3]
```diff
@@ -16,34 +16,34 @@ Here you will find guides mainly for Sonarr/Radarr/Bazarr and everything related
...
 
 ---
 
 ## Support
 
 [![Discord chat](https://img.shields.io/discord/492590071455940612?style=for-the-badge&color=4051B5&logo=discord){ .off-glb }](https://trash-guides.info/discord){:target="_blank" rel="noopener noreferrer"}
 
-If you have questions, suggestions, requests or want to contribute, click the chat badge to join the Discord Channel where you can ask your questions directly or use the following [link](https://github.com/TRaSH-Guides/Guides/issues){:target="_blank" rel="noopener noreferrer"} to open an issue on GitHub.
+If you have questions, suggestions, requests, or want to contribute, click the chat badge to join the Discord Channel. There, you can ask your questions directly. Alternatively, you can use the following [link](https://github.com/TRaSH-Guides/Guides/issues){:target="_blank" rel="noopener noreferrer"} to open an issue on GitHub.
```
Comment (zakkarry) [inline docs/index.md:30]:
```suggestion
If you have questions, suggestions, or requests, or want to contribute, click the chat badge to join the Discord Channel. There you can ask your questions directly, or you can use the following [link](https://github.com/TRaSH-Guides/Guides/issues){:target="_blank" rel="noopener noreferrer"} to open an issue on GitHub.

File: docs/index.md [hunk 4]
```diff
@@ -16,34 +16,34 @@ Here you will find guides mainly for Sonarr/Radarr/Bazarr and everything related
...
 Feel free to create a PR if you want to.
 
 ---
 
 ### DISCLAIMER
 
-The information I use for the guides is information I have gathered from different sources like support forums, Reddit or Discord, provided by user input or from the development teams.
-I have tested most things myself and I have tried to see if I could tune them a bit more to my needs and sometimes even for others.
+The information I use for the guides is information I have gathered from different sources like support forums, Reddit, or Discord, provided by user input or from the development teams.
+I have tested most things myself and I have tried to see if I could tune them a bit more to my needs (and sometimes even for others).
```
Comment (zakkarry) [inline docs/index.md:38]:
```suggestion
The information provided in the guides has been gathered from different sources such as support forums, Reddit, Discord, developed based on user input, or from the development teams themselves.
I have tested and tuned most things with the ultimate goal being to have them in line with my, and sometimes others', needs.

File: docs/index.md [hunk 5]
```diff
@@ -16,34 +16,34 @@ Here you will find guides mainly for Sonarr/Radarr/Bazarr and everything related
...
 
-You use everything here at your own risk. I won't be responsible if something happens to your setup or collection!
+You use everything here at your own risk. I am not responsible if something happens to your setup or collection!
```
Comment (zakkarry) [inline docs/index.md:40]:
```suggestion
All of the information provided in these guides, as well as in Discord or any guides-related mediums, is to be used at your own risk. I am not responsible if something happens to your setup or collection!

**Review by zakkarry — CHANGES_REQUESTED** [review]
Just my recommendations on language as @TRaSH- requested.

Feel free to discuss/comment on wording or grammar if you feel so inclined :)

**Comment by TRaSH-** [comment]
@OneRadRedDad do you agree with the suggested changes ?
i will add the suggestions in a batch and merge them together with the PR

**Comment by OneRadRedDad** [comment]
> @OneRadRedDad do you agree with the suggested changes ? i will add the suggestions in a batch and merge them together with the PR

I committed the ones I take no issue with. I'd like to propose some further changes to the remaining suggestions, but won't be able to do so for another 12-24 hours.

**Comment by OneRadRedDad** [comment]
@zakkarry I've made some slight tweaks to your remaining two suggestions. Let me know what you think.

**Review by zakkarry — APPROVED** [review]

**Comment by zakkarry** [comment]
lgtm

**Comment by zakkarry** [comment]
@TRaSH- merge whenever

**Pull request merged**